README.md

```diff
@@ -2,6 +2,8 @@
 
 This is Rakudo, a Raku Programming Language compiler for the MoarVM, JVM and Javascript virtual machines.
 
+NOTE: For those ambitious Raku coders see the [RakuAST](./RaluAST.md) project information.
```

Comment:
Similarly, I think this should be RakuAST.md

Reply:
Yes, good catch. Thanks. I’ll fix that.